tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -645,7 +652,8 @@ namespace Tensile
 
         if constexpr(insertKernelArgs)
             if(!internalArgsSupport.useUniversalArgs)
-                kernelArgs<T_Debug, true>(0, 0, args, problem.getParams());
+                kernelArgs<T_Debug, true>(
+                    0, (uint32_t)KERNELARGTYPE::NORMAL, args, problem.getParams());
```

Comment:
better to case at line#767, and set parameter type as KERNELARGTYPE at line#759.